<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QwcPrivateMessager</class>
 <widget class="QWidget" name="QwcPrivateMessager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="fMessageList"/>
   </item>
   <item row="0" column="1">
    <widget class="QTextBrowser" name="fMessageView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="fMessageInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
